app/measurement/management/commands/s3_query_export.py

```diff
@@ -60,7 +60,7 @@ class Command(BaseCommand):
             default: 'default'
 
     '''
-    BUCKET_NAME = 'squacapi-backup'
+    BUCKET_NAME = 'squacapi-measurements'
```

Comment:
These should be env variables too
SQUAC_MEASUREMENTS_BUCKET
SQUAC_AWS_REGION

Reply:
Would those go in the .env file or in settings.py? I see in settings there is already an AWS_DEFAULT_REGION defined from the AWS_DEFAULT_REGION env variable.

Reply:
Good catch, use settings.AWS_DEFAULT_REGION in this manager. Add another to settings.py constant called SQUAC_MEASUREMENTS_BUCKET that is set by a value in the .env file just like AWS_DEFAULT_REGION is